python/python/lancedb/remote/table.py

```diff
@@ -478,6 +479,27 @@ def compact_files(self, *_):
             "compact_files() is not supported on the LanceDB cloud"
         )
 
+    def optimize_indices(self, **kwargs):
+        """optimize_indices() is not supported on the LanceDB cloud.
+        Indices are optimized automatically."""
+        raise NotImplementedError(
+            "optimize_indices() is not supported on the LanceDB cloud. "
+            "Indices are optimized automatically."
+        )
```

Comment:
Why add an `optimize_indices` method here? It's not implemented and we don't have one for the local table?

Reply:
removed